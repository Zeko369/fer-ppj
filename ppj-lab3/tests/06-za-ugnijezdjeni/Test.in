<program>
 <lista_naredbi>
  <naredba>
   <naredba_pridruzivanja>
    IDN 1 x
    OP_PRIDRUZI 1 =
    <E>
     <T>
      <P>
       BROJ 1 2
      <T_lista>
       $
     <E_lista>
      $
  <lista_naredbi>
   <naredba>
    <za_petlja>
     KR_ZA 2 za
     IDN 2 i
     KR_OD 2 od
     <E>
      <T>
       <P>
        BROJ 2 1
       <T_lista>
        $
      <E_lista>
       $
     KR_DO 2 do
     <E>
      <T>
       <P>
        BROJ 2 5
       <T_lista>
        $
      <E_lista>
       $
     <lista_naredbi>
      <naredba>
       <za_petlja>
        KR_ZA 3 za
        IDN 3 j
        KR_OD 3 od
        <E>
         <T>
          <P>
           BROJ 3 1
          <T_lista>
           $
         <E_lista>
          $
        KR_DO 3 do
        <E>
         <T>
          <P>
           BROJ 3 5
          <T_lista>
           $
         <E_lista>
          $
        <lista_naredbi>
         <naredba>
          <za_petlja>
           KR_ZA 4 za
           IDN 4 k
           KR_OD 4 od
           <E>
            <T>
             <P>
              BROJ 4 1
             <T_lista>
              $
            <E_lista>
             $
           KR_DO 4 do
           <E>
            <T>
             <P>
              BROJ 4 5
             <T_lista>
              $
            <E_lista>
             $
           <lista_naredbi>
            <naredba>
             <za_petlja>
              KR_ZA 5 za
              IDN 5 l
              KR_OD 5 od
              <E>
               <T>
                <P>
                 BROJ 5 1
                <T_lista>
                 $
               <E_lista>
                $
              KR_DO 5 do
              <E>
               <T>
                <P>
                 BROJ 5 5
                <T_lista>
                 $
               <E_lista>
                $
              <lista_naredbi>
               <naredba>
                <za_petlja>
                 KR_ZA 6 za
                 IDN 6 m
                 KR_OD 6 od
                 <E>
                  <T>
                   <P>
                    BROJ 6 1
                   <T_lista>
                    $
                  <E_lista>
                   $
                 KR_DO 6 do
                 <E>
                  <T>
                   <P>
                    BROJ 6 5
                   <T_lista>
                    $
                  <E_lista>
                   $
                 <lista_naredbi>
                  <naredba>
                   <naredba_pridruzivanja>
                    IDN 7 x
                    OP_PRIDRUZI 7 =
                    <E>
                     <T>
                      <P>
                       IDN 7 i
                      <T_lista>
                       $
                     <E_lista>
                      OP_PLUS 7 +
                      <E>
                       <T>
                        <P>
                         IDN 7 j
                        <T_lista>
                         $
                       <E_lista>
                        OP_PLUS 7 +
                        <E>
                         <T>
                          <P>
                           IDN 7 k
                          <T_lista>
                           $
                         <E_lista>
                          OP_PLUS 7 +
                          <E>
                           <T>
                            <P>
                             IDN 7 l
                            <T_lista>
                             $
                           <E_lista>
                            OP_PLUS 7 +
                            <E>
                             <T>
                              <P>
                               IDN 7 m
                              <T_lista>
                               $
                             <E_lista>
                              $
                  <lista_naredbi>
                   $
                 KR_AZ 8 az
               <lista_naredbi>
                $
              KR_AZ 9 az
            <lista_naredbi>
             $
           KR_AZ 10 az
         <lista_naredbi>
          $
        KR_AZ 11 az
      <lista_naredbi>
       $
     KR_AZ 12 az
   <lista_naredbi>
    <naredba>
     <naredba_pridruzivanja>
      IDN 13 x
      OP_PRIDRUZI 13 =
      <E>
       <T>
        <P>
         IDN 13 x
        <T_lista>
         $
       <E_lista>
        OP_PLUS 13 +
        <E>
         <T>
          <P>
           BROJ 13 3
          <T_lista>
           $
         <E_lista>
          $
    <lista_naredbi>
     $
