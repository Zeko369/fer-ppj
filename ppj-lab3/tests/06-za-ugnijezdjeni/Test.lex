IDN 1 x
OP_PRIDRUZI 1 =
BROJ 1 2
KR_ZA 2 za
IDN 2 i
KR_OD 2 od
BROJ 2 1
KR_DO 2 do
BROJ 2 5
KR_ZA 3 za
IDN 3 j
KR_OD 3 od
BROJ 3 1
KR_DO 3 do
BROJ 3 5
KR_ZA 4 za
IDN 4 k
KR_OD 4 od
BROJ 4 1
KR_DO 4 do
BROJ 4 5
KR_ZA 5 za
IDN 5 l
KR_OD 5 od
BROJ 5 1
KR_DO 5 do
BROJ 5 5
KR_ZA 6 za
IDN 6 m
KR_OD 6 od
BROJ 6 1
KR_DO 6 do
BROJ 6 5
IDN 7 x
OP_PRIDRUZI 7 =
IDN 7 i
OP_PLUS 7 +
IDN 7 j
OP_PLUS 7 +
IDN 7 k
OP_PLUS 7 +
IDN 7 l
OP_PLUS 7 +
IDN 7 m
KR_AZ 8 az
KR_AZ 9 az
KR_AZ 10 az
KR_AZ 11 az
KR_AZ 12 az
IDN 13 x
OP_PRIDRUZI 13 =
IDN 13 x
OP_PLUS 13 +
BROJ 13 3
