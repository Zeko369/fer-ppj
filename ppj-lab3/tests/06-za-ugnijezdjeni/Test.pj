x = 2 
za i od 1 do 5
 za j od 1 do 5
  za k od 1 do 5
   za l od 1 do 5
    za m od 1 do 5
	 x = i + j + k + l + m
	az
   az
  az
 az 
az
x = x + 3
