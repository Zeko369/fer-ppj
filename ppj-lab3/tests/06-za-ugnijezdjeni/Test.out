7 2 i
7 3 j
7 4 k
7 5 l
7 6 m
13 1 x
